flutter/cpp/flutter/dart_backend_match.cc

```diff
@@ -8,9 +8,10 @@
 
 extern "C" struct dart_ffi_backend_match_result *dart_ffi_backend_match(
     const char *lib_path, const char *manufacturer, const char *model) {
+  LOG(INFO) << "checking backend '" << lib_path << "'...\n";
```

Comment:
On iOS, there is only TFLite as default backend and the log shows
`I flutter/cpp/flutter/dart_backend_match.cc:11] checking backend ''...`
`I flutter/cpp/flutter/dart_backend_match.cc:67] backend '' matches`
Should we check for this case and make the log more understandable?

Reply:
Fixed